src/templates/customer/nr_view_order/quote.template.html

```diff
@@ -1,36 +1,23 @@
-<!--#Start Column Left# ##-->
-<!--##[%load_template file:"column_left/template.html"%][%END load_template%]##-->
-<!--# End Column Left# ##-->
-
-
-
-<!--# Start Breadcrumb# ##-->
-<ul class="breadcrumb" aria-label="Breadcrumb"> 
-<li><a href="[@config:home_url@]">Home</a></li>
-<li><span class="divider">/</span> <a href="[%URL page:'account'%][%END URL%]">My Account</a></li>
-</ul>
-<!--# End Breadcrumb# ##-->
-<div class="page-header">
-<h1>Quote #[@order_id@]</h1>
+<div class="col-xs-12">
+	<ul class="breadcrumb" aria-label="Breadcrumb">
+		<li><a href="[@config:home_url@]">Home</a></li>
+		<li><a href="[%url page:'account'/%]">My Account</a></li>
+	</ul>
+	<div class="page-header">
+		<h1>View Quote</h1>
+	</div>
+	<script language="javascript" type="text/javascript">
+		function print_quote() {
+			var win = window.open('[%url page:'account' type:'print' fn:'invoice' id:'[@order_id@]'/%]', 'print_invoice', 'width=740,height=400,scrollbars=yes,resizeable=yes');
+		}
+	</script>
+	<div class="row">
+		<div class="col-xs-12">
+			<input type="button" class="btn btn-default pull-left" value="Print Quote" onClick="print_quote()">&nbsp;
```

Comment:
Place icon back in please - <i class="fa fa-check icon-white"></i>

Reply:
Actually don't worry, if it's not in the invoice one then let's keep it consistent.

Reply:
@philcon93 actually you raise a good point though... Should the print button have a printer icon? 
![image](https://user-images.githubusercontent.com/6867163/37446228-b091cc0e-2867-11e8-86d6-186ebfef1f57.png)

The button on the my account page does:
![image](https://user-images.githubusercontent.com/6867163/37446243-c3e935bc-2867-11e8-87a8-1e60d4d6e068.png)